pkg/cmd/convert.go

```diff
@@ -62,15 +69,39 @@ func GetValidatedObject(v *viper.Viper, cmd *cobra.Command, out, outerr io.Write
 	}
 
 	var ocObjects []*object.OpenCompose
+	var data []byte
```

Comment:
Oh wait, I put it outside the loop, it should be inside the loop out the if, else; sorry got a bit confused :)